### PR TITLE
fix: (*RowsEvent).handleUnsigned()  panic

### PR DESCRIPTION
**Pull request by equnchen**
canal_panic

### CODE REVIEW

**Comment by lance6716** [comment]
Hi, can you add an unit test for it?

**Comment by equnchen** [comment]
> Hi, can you add an unit test for it?

Sorry, I'm very busy at work recently. How about we fix this obvious bug first?

**Comment by lance6716** [comment]
we have other fix PR like https://github.com/go-mysql-org/go-mysql/pull/495 https://github.com/go-mysql-org/go-mysql/pull/689 https://github.com/go-mysql-org/go-mysql/pull/792 . In order to be fair to them I also request the unit test for this PR

File: canal/rows_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+			r := &RowsEvent{
+				Table:  tt.fields.Table,
+				Action: tt.fields.Action,
+				Rows:   tt.fields.Rows,
+				Header: tt.fields.Header,
+			}
+			r.handleUnsigned()
+			if !reflect.DeepEqual(tt.fields.Rows, tt.wantRows) {
+				t.Errorf("rows=%+v, wantRows=%+v, not equal", tt.fields.Rows, tt.wantRows)
+			}
```
Comment (lance6716) [inline canal/rows_test.go:58]:
```suggestion
			require.Equal(t, tt.wantRows, r.Rows)
```

also import "github.com/stretchr/testify/require"

**Review by lance6716 — COMMENTED** [review]
rest lgtm. Thank you

File: canal/rows_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package canal
+
+import (
```
Comment (lance6716) [inline canal/rows_test.go:3]:
```
import (
	"testing"

	"github.com/go-mysql-org/go-mysql/replication"
	"github.com/go-mysql-org/go-mysql/schema"
	"github.com/stretchr/testify/require"
)
```

This should pass CI

File: canal/rows_test.go (hunk 1 above)
Comment (equnchen) [reply canal/rows_test.go]: done

File: canal/rows_test.go (hunk 2 above)
Comment (equnchen) [reply canal/rows_test.go:3]: done

**Pull request merged**